x/cellarfees/types/params.go

```diff
@@ -1,10 +1,21 @@
 package types
 
 import (
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	"gopkg.in/yaml.v2"
 )
 
+// Parameter keys
+var (
+	KeyAuctionBlockDelay          = []byte("auctionblockdelay")
+	KeyRewardEmissionPeriod       = []byte("rewardemissionperiod")
+	KeyInitialPriceDecreaseRate   = []byte("initialpricedecreaserate")
+	KeyPriceDecreaseBlockInterval = []byte("pricedecreaseblockinterval")
+)
```

Comment:
nit: camel case might make these easier to read

Reply:
Could these also be `const` or is there an issue setting them using the `[]byte` definition method?